scrape.py

```diff
@@ -363,7 +365,7 @@ def generate_location_steps3(num_steps):
 
 
 def generate_location_steps4(loc, num_steps):
-    meters_delta = 150
+    meters_delta = RADIUS * math.sqrt(3) - 1
```

Comment:
-1 is to force some amount of overlap (to counteract the estimation with the other constants such as 111111)